src/components/DirectPaymentCreateModal/DirectPaymentCreateModal.tsx

```diff
@@ -286,142 +309,265 @@ export const DirectPaymentCreateModal: React.FC<DirectPaymentCreateModalProps> =
     });
   };
 
+  const renderConfirmationScreen = () => {
+    if (!paymentDataToConfirm) return null;
+
+    // Helper function to shorten wallet addresses
+    const shortenAddress = (
+      address: string,
+      startChars: number = 6,
+      endChars: number = 8,
+    ): string => {
+      if (!address || address.length <= startChars + endChars + 3) return address;
+      return `${address.slice(0, startChars)}...${address.slice(-endChars)}`;
+    };
+
+    const isWalletAddress = paymentDataToConfirm.receiver.wallet_address;
+    const receiverInfo = isWalletAddress
+      ? `Wallet Address: ${shortenAddress(paymentDataToConfirm.receiver.wallet_address || "")}`
+      : formData.selectedReceiver
+        ? getReceiverDisplayInfo(formData.selectedReceiver, formData.receiverSearch)
+        : "Unknown Receiver";
+
+    // Get the selected wallet address for non-wallet address payments
+    const selectedWallet =
+      !isWalletAddress && formData.selectedReceiver && formData.walletId
+        ? formData.selectedReceiver.wallets.find((w) => w.wallet.id === formData.walletId)
+        : null;
+    const walletAddress = selectedWallet?.stellar_address || paymentDataToConfirm.wallet?.address;
+
+    return (
+      <div className="DirectPaymentCreateModal__confirmation">
+        <div className="DirectPaymentCreateModal__confirmationContent">
+          <div className="DirectPaymentCreateModal__confirmationWarning">
+            <p>Please review the payment details below. This action cannot be undone.</p>
+          </div>
```

Comment:
We can use `Alert` component from SDS here ([docs](https://design-system.stellar.org/alert)). The message is using color CSS variable that no longer exist. The old SDS had prefix `--color`, the new has `--sds-clr-`.

<img width="523" height="313" alt="image" src="https://github.com/user-attachments/assets/0033d1d3-60a0-48d3-a68e-995a039bee09" />

<img width="742" height="651" alt="image" src="https://github.com/user-attachments/assets/a4501d35-4d39-40fb-bf51-23f035e155fa" />